ksql-metastore/src/main/java/io/confluent/ksql/metastore/MetaStoreImpl.java

```diff
@@ -154,56 +156,55 @@ public void updateForPersistentQuery(final String queryId,
   }
 
   private void addSourceNames(final Set<String> sourceNames, final String queryId) {
-    for (final String sourceName: sourceNames) {
-      final ReferentialIntegrityTableEntry referentialIntegrity =
-          dataSourceMap.get(sourceName).referentialIntegrity;
-
-      referentialIntegrity.addSourceForQueries(queryId);
-    }
+    sourceNames.forEach(sourceName -> {
```

Comment:
Yep, I agree. I generally would of left a forEach like this alone. I think this was likely a artefact of refactoring, i.e. I'll of converted the forEach to a stream call because I wanted to use streams, then later removed some of the operations, so it's back to just a forEach, and then not realised.

Call this kind of thing out is fine by me.  In the new code this function doesn't exist, so its not an issue ;)